### ISSUE TITLE
Unable to add db password using a file in secrets

### ISSUE THREAD

**Issue by LBF38** [docker/docs#18204]
### Is this a docs issue?

- [X] My issue is about the documentation content or website

### Type of issue

Information is incorrect

### Description

As the documentation says, when using Docker secrets in Compose, the entire content of the file is set as a secret.
However, when I'm following the instructions from the NodeJS Docker example, I have trouble with setting up the db password.

I managed to make it work with the `POSTGRES_PASSWORD `env variable inside the Compose file. But I didn't manage to make it work with a file as a secret. It seems that there is the return character at the end of the file which destroys everything.

Thus, is it possible to mention an environment file containing `POSTGRES_PASSWORD` as an env variable in the secrets definition ?
Would it be secure enough ? What should be done to go this way ?

### Location

https://docs.docker.com/language/nodejs/develop/

### Suggestion

I would suggest to have a linked `.env` files for secrets definition, maybe ? or a `secrets.env`, I don't know.

**Comment by craig-osterhout** [comment]
Thanks for sharing the issue. 
To learn more about why a secret is preferred over an environment variable, please see https://docs.docker.com/compose/use-secrets/


If possible, can you share what operating system you're using and any details about how you created the password.txt file?

**Comment by LBF38** [comment]
Thanks for replying !

I have found a way to make it work by using an env variable as a secret from [this example](https://docs.docker.com/compose/compose-file/09-secrets/#example-2).
But when using the file with the password in it (`.txt`), the content has the CRLF/LF as docker uses the whole file content, if I have correctly understood.

I'm using Windows for this Docker example. I have also tested on WSL. Both via VS Code or via CLI.
Here what I have tested: 
- On Windows: 
   - via VS Code new file command. => `.txt` file with the password in it. would have CRLF EOF
   - via PowerShell using the command: `echo "password" > password.txt`
- On WSL:
   - via VS Code new file command. same as Windows but this time would have LF EOF.
   - via Bash using the same command as above.

Hope it helps improve the docs !
